backend/LeetWars.Core/LeetWars.Core.WebAPI/Extentions/ServiceCollectionExtensions.cs

```diff
@@ -27,6 +27,7 @@ public static void RegisterCustomServices(this IServiceCollection services, ICon
 
             services.AddScoped<IMessageSenderService, MessageSenderService>();
             services.AddTransient<IChallengeService, ChallengeService>();
+            services.AddTransient<IChallengeLevelService, ChallengeLevelService>();
             services.AddTransient<ITagService, TagService>();
             services.AddTransient<ILanguageService, LanguageService>();
```

Comment:
I guess these shouldn't be transient

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/106184656/5955285e-7e1c-4325-9e61-3b2f22ce60b3)